events/bills/whmcs.go

```diff
@@ -57,18 +62,19 @@ func (w whmcsBiller) Onboard(o *BillOpts, m map[string]string) error {
 
 	client := whmcs.NewClient(nil, m[constants.DOMAIN])
 	a := map[string]string{
-		"username":    m[constants.USERNAME],
-		"password":    GetMD5Hash(m[constants.PASSWORD]),
-		"firstname":   acc.FirstName,
-		"lastname":    acc.FirstName,
-		"email":       acc.Email,
-		"address1":    "Dummy address",
-		"city":        "Dummy city",
-		"state":       "Dummy state",
-		"postcode":    "00001",
-		"country":     "IN",
-		"phonenumber": "981999000",
-		"password2":   string(sDec),
+		"username":     m[constants.USERNAME],
+		"password":     GetMD5Hash(m[constants.PASSWORD]),
+		"firstname":    acc.FirstName,
+		"lastname":     acc.FirstName,
+		"email":        acc.Email,
+		"address1":     "Dummy address",
+		"city":         "Dummy city",
+		"state":        "Dummy state",
+		"postcode":     "00001",
+		"country":      "IN",
+		"phonenumber":  "981999000",
+		"password2":    string(sDec),
+		"customfields": GetBase64(map[string]string{"vertice_email": acc.Email, "vertice_apikey": acc.ApiKey}),
```

Comment:
@ranjithamca  Can you make 
two constants
VERTICE_EMAIL="vertice_email"
VERTICE_APIKEY="vertice_apikey"